### ISSUE TITLE
ParallaxImage cannot change property: 'resizeMode'

### ISSUE THREAD

**Issue by SnowDang** [archriss/react-native-snap-carousel#151]
I want to change 'resizeMode' of ParallaxImage but I can not.
After set style={{resizeMode: 'contain'}} does not working for me.
                    <ParallaxImage
                        source={{ uri: item.thumbnails.medium.url}}
                        containerStyle={[Styles.imageContainer]}
                        style={{resizeMode : 'contain'}}
                        showSpinner={true}
                        {...parallaxProps}
                    />
How can I change it? Please help me?

**Comment by bd-arc** [comment]
Hi @SnowDang,

If you take a look at [this line of code](https://github.com/archriss/react-native-snap-carousel/blob/master/src/parallaximage/ParallaxImage.js#L177), you'll see that component's style take precedence over yours. I did that in order to prevent potential mistakes, particularly setting the `position` to something other than `relative` (which would completely mess with the parallax effect).

I will shortly push a fix that will allow you to pass your custom styles.

**Comment by bd-arc** [comment]
And [here you are](https://github.com/archriss/react-native-snap-carousel/commit/b4b31048114b1555c4d1f893db12b60bd40a16e1)!

If you want to use this commit right away, you can modify your `package.json` file this way:
```
"react-native-snap-carousel": "https://github.com/archriss/react-native-snap-carousel#b4b3104
```

Can you confirm that it works as expected?

**Comment by SnowDang** [comment]
Thank @bd-arc very much.
I can modify it with your way.

**Comment by bd-arc** [comment]
@SnowDang The fix is available in version `3.2.2` ;-)
